script/github-actions/select-unit-tests.js

```diff
@@ -12,21 +13,24 @@ const DISALLOWED_SPECS = ALLOW_LIST.filter(spec => spec.allowed === false).map(
   spec => spec.spec_path.substring(spec.spec_path.indexOf('src')),
 );
 
-const CHANGED_APPS = process.env.CHANGED_FILES
-  ? process.env.CHANGED_FILES.split(' ').map(filePath =>
-      filePath
-        .split('/')
-        .slice(0, 3)
-        .join('/'),
-    )
-  : [];
+const CHANGED_APPS =
+  CHANGED_FILES !== []
```

Comment:
I think this needs to be using a `length` check - JS can't compare arrays like this

Reply:
Odd - seemed to work alright, but changed it just in case. I do agree checking for a length is more dependable in general.